src/pages/docs/tutorials/tutorial-kitten-browser/data-and-state/index.md

```diff
@@ -127,21 +129,20 @@ computed: {
 	}
 },
 
-render: ({children, onSelect, url, ...rest}) => {
+render: ({children, onSelect, size, url, ...rest}) => {
 	delete rest.index;
-	delete rest.size;
 
 	return (
 		<div {...rest} onClick={onSelect}>
-			<img src={url} />
+			<img src={url} alt="Kitten" width={size} height={size} />
 			<div>{children}</div>
 		</div>
 	);
 }
 ```
 ### Adding Spotlight Support
 
-In [Hello, Enact!](../../tutorial-hello-enact/), we [introduced `MoonstoneDecorator`](../../tutorial-hello-enact/adding-moonstone-support/), which adds the base support for Spotlight in an application. All of our Moonstone controls that should be spottable support Spotlight out of the box. If you're creating a custom component, like we have in this example, you'll have to add that support yourself. Fortunately, in most cases, you can add Spotlight support by wrapping your component with the `Spottable` HOC.
+In [Hello, Enact!](../../tutorial-hello-enact/), we [introduced `ThemeDecorator`](../../tutorial-hello-enact/adding-moonstone-support/), which adds the base support for Spotlight in an application. All of our Sandstone controls that should be spottable support Spotlight out of the box. If you're creating a custom component, like we have in this example, you'll have to add that support yourself. Fortunately, in most cases, you can add Spotlight support by wrapping your component with the `Spottable` HOC.
```

Comment:
I'm seeing `adding-moonstone-support`, is this correct?

Reply:
That's not what I intended. I missed it. 
Thank you.

Fixed in latest commit (to "adding-sandstone-support")